### PR TITLE
Localisation fixes to Dutch/Polish

### PR DESCRIPTION
**Pull request by RedDevilus**
https://github.com/PCSX2/pcsx2/commit/2c3973a9b262579d01eab35193052cc3a6f82109 is my changes

Let's just say this is a mess. Failed rebase :/

### CODE REVIEW

**Comment by RedDevilus** [comment]
Ok i finally merged it. Sorry for this mess. Let's just say this is not good practise. You can check the files for review.

File: locales/pl_PL/pcsx2_Main.po [hunk 1]
```diff
@@ -1221,14 +1221,12 @@ msgid "Welcome to PCSX2!"
...
-#, fuzzy
 msgid "Configuration Guide"
-msgstr "Poradniki Konfiguracji (Online)"
+msgstr "Poradniki konfiguracji"
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:110
-#, fuzzy
 msgid "Readme / FAQ"
-msgstr "Readme / FAQ (Offline/PDF)"
+msgstr "PRZECZYTAJ MNIE / NAJCZEŚCIEJ ZADAWANE PYTANIA"
```
Comment (lightningterror) [inline locales/pl_PL/pcsx2_Main.po]: It doesn't need to be in caps.

Comment (RedDevilus) [reply locales/pl_PL/pcsx2_Main.po]: Okidoki ez fix

File: locales/pl_PL/pcsx2_Main.po [hunk 2]
```diff
@@ -1221,14 +1221,12 @@ msgid "Welcome to PCSX2!"
 msgstr "Witaj w PCSX2!"
 
 #: pcsx2/gui/Dialogs/FirstTimeWizard.cpp:106
-#, fuzzy
 msgid "Configuration Guide"
-msgstr "Poradniki Konfiguracji (Online)"
+msgstr "Poradniki konfiguracji"
```
Comment (lightningterror) [inline locales/pl_PL/pcsx2_Main.po]: Konfiguracji with capital letter.

**Comment by RedDevilus** [comment]
Ok fixes are done.

File: locales/pl_PL/pcsx2_Main.po (hunk 2 above)
Comment (mavethee) [reply locales/pl_PL/pcsx2_Main.po]: @lightningterror well 🤔

**Pull request merged**